draft-ietf-tls-esni.md

```diff
@@ -215,14 +215,19 @@ The ECH configuration is defined by the following `ECHConfig` structure.
     struct {
         HpkeKdfId kdf_id;
         HpkeAeadId aead_id;
-    } ECHCipherSuite;
+    } HpkeSymmetricCipherSuite;
 
     struct {
-        opaque public_name<1..2^16-1>;
-        HpkePublicKey public_key;
+        opaque config_id<0..255>;
         HpkeKemId kem_id;
-        ECHCipherSuite cipher_suites<4..2^16-4>;
+        HpkePublicKey public_key;
+        HpkeSymmetricCipherSuite cipher_suites<4..2^16-4>;
+    } HpkeKeyConfig;
+
+    struct {
+        HpkeKeyConfig key_config;
```

Comment:
It's a little goofy to have the config_id apply to the whole thing here, but I suppose it's inlined.

Reply:
I think we at least need a comment here to explain that.

Reply:
Can you suggest text?